catalogs/catalog-jdbc-doris/src/test/java/com/datastrato/gravitino/catalog/doris/integration/test/DorisTableOperationsIT.java

```diff
@@ -406,4 +413,45 @@ public void testCreateNotSupportTypeTable() {
                       "Couldn't convert Gravitino type %s to Doris type", type.simpleString())));
     }
   }
+
+  @Test
+  void testSortOrderTable() {
+    String tableName = GravitinoITUtils.genRandomName("doris_sort_order_table");
+
+    String tableComment = "test_comment";
+    List<JdbcColumn> columns = new ArrayList<>();
+    JdbcColumn col_1 =
+        JdbcColumn.builder().withName("col_1").withType(INT).withComment("id").build();
+    columns.add(col_1);
+    JdbcColumn col_2 =
+        JdbcColumn.builder().withName("col_2").withType(VARCHAR_255).withComment("col_2").build();
+    columns.add(col_2);
+    JdbcColumn col_3 =
+        JdbcColumn.builder().withName("col_3").withType(VARCHAR_255).withComment("col_3").build();
+    columns.add(col_3);
+
+    Distribution distribution = Distributions.hash(32, NamedReference.field("col_1"));
+    Index[] indexes = new Index[] {};
+
+    SortOrder[] sortOrders =
+        new SortOrder[] {
+          SortOrders.of(NamedReference.field("col_1"), SortDirection.ASCENDING),
```

Comment:
Doris DUPLICATE key seems to not support SortDirection, should we use another concept to pass this parameter?

Reply:
Yes, It will use the default sort direction `ASCENDING`.